hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconServer.java

```diff
@@ -234,8 +242,13 @@ public StorageContainerServiceProvider getStorageContainerServiceProvider() {
   }
 
   @VisibleForTesting
-  public ContainerDBServiceProvider getContainerDBServiceProvider() {
-    return containerDBServiceProvider;
+  public ReconContainerMetadataManager getContainerDBServiceProvider() {
+    return reconContainerMetadataManager;
+  }
+
+  @VisibleForTesting
+  public ReconNamespaceSummaryManager getReconNamespaceSummaryManager() {
```

Comment:
yeah sure, but the next step will be adding a new -du task to query info from OM and write into RDB, so maybe it could be used for testing that. But for now, I am good with removing it.